ceaos/sensors/nwsensor.py

```diff
@@ -0,0 +1,26 @@
+from .sensor_definition import Sensor
+from ..loggers.InfluxDB import InfluxDBLogger
+
+
+class Network_Sensor(Sensor):
+    def __init__(self, sensor=None, ):
+        self.curr_value = None
+        self.logger = InfluxDBLogger()
+        # I think each Network_Sensor is essentially
+        # attached to an actual sensor
+        self.sensor = sensor
+
+    def read_value(self):
+        return self.sensor.read_value()
```

Comment:
Just store the value inside the object

Reply:
what do you mean by that? isn't each network sensor associated with a "real sensor" so the network sensor read_value is essentially just calling another sensors read value? Or by store do you mean create a attribute that also stores the value and returns it?

Reply:
No need for a real sensor, just a value attribute is fine